metrics/density/README.md

```diff
@@ -0,0 +1,46 @@
+# Kata Containers density metrics tests
+
+This directory contains a number of tests to help measure container
+density. Some measures are based around the PSS of the runtime components,
+and others look at the system level ('free' and '/proc/meminfo' for instance)
+impact.
+
+## docker_memory_usage
+
+This test measures the PSS footprint of the runtime components whilst
+launching a number of small (busybox) containers.
+
+## fast_footprint
+
+This test takes a number of system level measures after launching a number
```

Comment:
The double "number" sounds slightly odd to me. How about a slight tweak to something like:

> This test takes gathers various system level details after launching a number of containers in parallel ...